liblepton/src/g_rc.c

```diff
@@ -760,8 +760,8 @@ SCM g_rc_reset_component_library(void)
 SCM g_rc_attribute_promotion(SCM mode)
 {
   static const vstbl_entry mode_table[] = {
-    {TRUE , "enabled" },
-    {FALSE, "disabled"},
+    {TRUE , (char*) "enabled" },
```

Comment:
I think it's best to change the definition of the `struct vstbl_entry` so that it uses `const char*` rather than `char*`.